infra/kubernetes/main.tf

```diff
@@ -0,0 +1,81 @@
+// terraform config //
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+  }
+  # backend "s3" {
+  #   bucket  = "cncf-credits-infra-kubernetes-tfstate"
+  #   key     = "terraform.tfstate"
+  #   region  = "ap-southeast-2"
+  #   encrypt = true
+  # }
+}
+
+provider "aws" {
+  profile = "default"
+  region  = "ap-southeast-2"
+}
+// ---------------- //
+
+// the tfstate for the management of the kubernetes account+OU //
+resource "aws_kms_key" "cncf-credits-infra-kubernetes-tfstate" {
+  description = "This key is used to encrypt bucket objects"
+}
+resource "aws_s3_bucket" "cncf-credits-infra-kubernetes-tfstate" {
```

Comment:
For the moment, it's not a requirement and any thing related to S3 bucket creation

Reply:
The kms key is used for encryption on the s3 bucket
https://github.com/cncf/credits/pull/3/files#diff-1cf1dfd6d8ea7f7e0e0a4a572ec241e22a3b30607622b2fa17febed09baeafedR41